Mathlib/Data/ENat/Basic.lean

```diff
@@ -261,6 +261,17 @@ theorem nat_induction {P : ℕ∞ → Prop} (a : ℕ∞) (h0 : P 0) (hsuc : ∀
   · exact htop A
   · exact A _
 
+lemma add_one_pos : 0 < n + 1 :=
+  succ_def n ▸ Order.bot_lt_succ n
+
+lemma add_lt_add_iff_right {k : ℕ∞} (h : k ≠ ⊤) : n + k < m + k ↔ n < m :=
+  WithTop.add_lt_add_iff_right h
+
+lemma add_lt_add_iff_left {k : ℕ∞} (h : k ≠ ⊤) : k + n < k + m ↔ n < m :=
+  WithTop.add_lt_add_iff_left h
+
+section withTop_enat
```

Comment:
Section names aren't all that important, but I think simply `WithTop` would be a better namespace here, as it's already in the `ENat` namespace

Reply:
I'd rather keep `withTop_enat`: otherwise, the section could be interpreted as a discussion on the relationship between `ENat` and `WithTop Nat`, and ways to go back and forth.